CHANGELOG.md

```diff
@@ -14,6 +14,9 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 - ``cartridge pack`` generates ``VERSION.lua`` file with the current
   version of project.
 
+- Ability to specify fd limit in the systemd unit template
+  (command ``cartridge pack``), using the ``--unit-params-file`` flag.
```

Comment:
Ability to specify fd limit in the systemd unit template
  (command ``cartridge pack``) in the ``systemd-unit-params.yml`` file?